commit 7fc9152e831fb24091c0ceabdcecb9b07dd29dd6
Author: H.J. Lu <hjl.tools@gmail.com>
Date:   Wed Apr 21 18:40:08 2021 -0700

    x86: tst-cpu-features-supports.c: Update AMX check
    
    Pass "amx-bf16", "amx-int8" and "amx-tile", instead of "amx_bf16",
    "amx_int8" and "amx_tile", to __builtin_cpu_supports for GCC 11.

diff --git a/sysdeps/x86/tst-cpu-features-supports.c b/sysdeps/x86/tst-cpu-features-supports.c
index ce78a7d8bcce75a4..a2cabc90beeb9530 100644
--- a/sysdeps/x86/tst-cpu-features-supports.c
+++ b/sysdeps/x86/tst-cpu-features-supports.c
@@ -59,9 +59,9 @@ do_test (int argc, char **argv)
   fails += CHECK_SUPPORTS (aes, AES);
 #endif
 #if __GNUC_PREREQ (11, 1)
-  fails += CHECK_SUPPORTS (amx_bf16, AMX_BF16);
-  fails += CHECK_SUPPORTS (amx_int8, AMX_INT8);
-  fails += CHECK_SUPPORTS (amx_tile, AMX_TILE);
+  fails += CHECK_SUPPORTS (amx-bf16, AMX_BF16);
+  fails += CHECK_SUPPORTS (amx-int8, AMX_INT8);
+  fails += CHECK_SUPPORTS (amx-tile, AMX_TILE);
 #endif
   fails += CHECK_SUPPORTS (avx, AVX);
   fails += CHECK_SUPPORTS (avx2, AVX2);
